src/coverage.ts

```diff
@@ -42,5 +42,7 @@ export default async (options: IOptions) => {
 
   return Promise.all(promises).then(() => {
```

Comment:
I think this might be simplified just using async await, since it's already in use. 
```suggestion
  const results = await Promise.all(promises);
  options.reporter(results, options.schema);
```

Reply:
oh yea nice